src/test/java/net/sf/jabref/logic/layout/format/LatexToUnicodeFormatterTest.java

```diff
@@ -61,4 +61,16 @@ public void testEquations() {
     public void formatExample() {
         assertEquals("Mönch", formatter.format(formatter.getExampleInput()));
     }
+
+    @Test
+    public void testEarlierProblems() {
```

Comment:
Please split tests so that the contain only one assertEquals, also `convertChi` is probably a better name (you can also reference the issue to give some context)